test/test_browser.py

```diff
@@ -5536,19 +5547,30 @@ def test_error_reporting(self):
 
   @also_with_threads
   @parameterized({
-    '': (False,),
-    'es6': (True,),
+    '': (False,False),
+    'blob': (False,True),
+    'es6': (True,False),
+    'es6_blob': (True,True)
   })
-  def test_webpack(self, es6):
-    if es6:
+  def test_webpack(self, es6, useblob):
+    if es6 and useblob:
+      copytree(test_file('webpack_es6_with_blob'), '.')
+      self.emcc_args += ['-sEXPORT_ES6', '-pthread', '-sPTHREAD_POOL_SIZE=1']
+      outfile = 'src/hello.mjs'
+    elif es6:
       copytree(test_file('webpack_es6'), '.')
       self.emcc_args += ['-sEXPORT_ES6', '-pthread', '-sPTHREAD_POOL_SIZE=1']
       outfile = 'src/hello.mjs'
+    elif useblob:
+      copytree(test_file('webpack_with_blob'), '.')
+      self.emcc_args += ['-pthread', '-sPTHREAD_POOL_SIZE=1']
+      outfile = 'src/hello.js'
     else:
       copytree(test_file('webpack'), '.')
       outfile = 'src/hello.js'
     self.compile_btest('hello_world.c', ['-sEXIT_RUNTIME', '-sMODULARIZE', '-sENVIRONMENT=web,worker', '-o', outfile])
     self.run_process(shared.get_npm_cmd('webpack') + ['--mode=development', '--no-devtool'])
+    shutil.copy(outfile, 'dist/')
```

Comment:
I think the idea is that the bundler should do this.. so you should not have to do this here.  In fact, if you have to do this that would seem like a bug maybe?   (Same with the .wasm file below I guess).

In any case, I think this bundler stuff is orthogonal the this change which I think we can test without modifying any bundler tests.

Reply:
Sorry, this one wasn't meant to be included here.